CONTRIBUTING.md

```diff
@@ -1,80 +1,96 @@
+- Any pull request can be removed for violation of these guidelines.
 
-The project coding conventions in [doc/developer-notes.md](doc/developer-notes.md) must be adhered to.
+Github committers do the actual merging of pull requests and create the official releases. If voters change their mind, committers will un-merge the pull request or reopen a pull request. Please remember to read through pull requests before submitting your own to avoid conflicting requests. 
 
-In general [commits should be atomic](https://en.wikipedia.org/wiki/Atomic_commit#Atomic_commit_convention) and diffs should be easy to read. For this reason do not mix any formatting fixes or code moves with actual code changes.
+Issues are to be submitted [here](https://github.com/bitcoinclassic/bitcoinclassic/issues).
 
-Commit messages should be verbose by default consisting of a short subject line (50 chars max), a blank line and detailed explanatory text as separate paragraph(s); unless the title alone is self-explanatory (like "Corrected typo in main.cpp") then a single title line is sufficient. Commit messages should be helpful to people reading your code in the future, so explain the reasoning for your decisions. Further explanation [here](http://chris.beams.io/posts/git-commit/).
+## <a name="communication"></a> Communication
+Bitcoin Classic is designed to allow users, miners, and developers to express their opinions equally and transparently. To do so, multiple channels of communication have been set up, see below.
 
-If a particular commit references another issue, please add the reference, for example "refs #1234", or "fixes #4321". Using "fixes or closes" keywords will cause the corresponding issue to be closed when the pull request is merged.
+- [Slack Channel](http://invite.bitcoinclassic.com/)
+- [Subreddit](https://www.reddit.com/r/bitcoin_classic)
+- [Consider.it](https://bitcoinclassic.consider.it/)
 
-Please refer to the [Git manual](https://git-scm.com/doc) for more information about Git.
+## <a name="merge-window"></a> Merge Window
+Using merge windows allows Bitcoin Classic to remain focused on specific issues at hand without distraction. A merge window decides the focus of development for the continuation of that window. In this window, code which is deemed to be sufficiently stable (and which is accepted by the community) is merged. Merge windows will be closed based on a vote via [consider.it](https://bitcoinclassic.consider.it/) and another vote will be held to open the next merge window. For details on the current merge window see below.
 
-  - Push changes to your fork
-  - Create pull request
+ - `Hard fork bitcoin to a 2 MB blocksize limit.`
+ - `2 MB Test Net`
+ - `Consensus Rule changes - O(n^2) protections`
+ - `Activication / versionbits code`
```

Comment:
Eventually we will want to replace this text with a link to some webpage that shows our current roadmap and merge window schedule. This is fine for now.